my_project/extract_df.py

```diff
@@ -362,8 +362,6 @@ def visibility(df, name):
     df[name] = df[name] * 0.6215
 
 
-def humidity(df, name):
-    df[name] = df[name] * 0.0624
```

Comment:
Because the unit for humidity ratio is changed to 'mess/mess', the conversion function is not needed.